pkgs/tools/security/metasploit/gemset.nix

```diff
@@ -1401,4 +1401,4 @@
     };
     version = "2.4.2";
   };
-}
+}
```

Comment:
Please add that back.

Reply:
Hmmm, that's wired...`update.sh` is generating this file and the script was not touched for months.

Reply:
Ah, #110395 changed the file.

Reply:
`update.sh` updated